### DISCUSSION TITLE
Temas Controvertidos en Informática

### DISCUSSION THREAD

**Discussion by shone900** [community/community#147018] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Los [temas controvertidos en informática ](https://techncleo.com/temas-controviertes-en-informatica/)reflejan el impacto profundo que la tecnología tiene en nuestra sociedad. Aspectos como la privacidad de los datos, la inteligencia artificial y la automatización generan debates sobre ética, seguridad y equidad. Además, cuestiones como el uso indebido de la información, la ciberseguridad y el acceso desigual a la tecnología plantean desafíos críticos. Es fundamental fomentar un diálogo abierto y responsable para equilibrar los avances tecnológicos con los valores humanos y garantizar un desarrollo inclusivo y sostenible.

**Comment by shinybrightstar** [comment]
Hi @shone900 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
